src/Chunk.ts

```diff
@@ -254,6 +255,12 @@ export default class Chunk {
 			}
 		}
 		this.suggestedVariableName = makeLegal(this.generateVariableName());
+		if (this.outputOptions.sanitizeFileName === false)
+			this.sanitizeFileName = (id) => id;
+		else if (typeof this.outputOptions.sanitizeFileName === 'function')
+			this.sanitizeFileName = this.outputOptions.sanitizeFileName;
+		else
+			this.sanitizeFileName = sanitizeFileName;
```

Comment:
As you are basically doing option normalisation here, I think it would be nicer to move this to `normalizeOutputOptions.ts` and just have the `sanitizeFileName` as a function in the normalised output options type. This would also avoid determining the default separately for each chunk. Also I would not copy over an option that is not chunk specific but rather directly access it from `this.outputOptions` to reduce the number of properties put on `Chunk`.